episodes/24-collaborating.md

```diff
@@ -218,7 +218,15 @@ the Reviewer should follow all the steps but close the PR at the end instead of
 Having an open, publicly-visible list of all the issues with your project is a helpful way of letting people know you are aware of issues and you are working on them. This can indicate to an external audience that the project is active.
 It also provides you and your collaborators with an "at a glance" view of the state of the project, making it easier to prioritise future work.
 
-As we have seen in the previous episode, GitHub's notifications framework **Mentions** plays an important part in communicating between collaborators and is used as a way of alerting team members of activities and referencing one issue/comment/pull requests from another.
+GitHub provides a useful notification feature for collaborative work - _mentions_. 
+The mention system notifies team members when somebody else references them 
+in an issue, comment or pull request - 
```

Comment:
It looks like what you want — is [an emdash](https://www.merriam-webster.com/words-at-play/em-dash-en-dash-how-to-use)

In general, I go by this rubric:

compound word? hyphen
numbers? endash
antici — pation? emdash

```suggestion
GitHub provides a useful notification feature for collaborative work --- _mentions_. 
The mention system notifies team members when somebody else references them 
in an issue, comment or pull request --- 
```

Reply:
I'm leaving them here for now, but have opened #218 to ensure they get addressed throughout the whole lesson at in future.